qiling/os/windows/dlls/ntdll.py

```diff
@@ -68,6 +70,27 @@ def _QueryInformationProcess(ql: Qiling, address: int, params):
 
         res_data = bytes(pci_obj)
 
+    
+    elif flag == ProcessCookie:
+        hCurrentProcess = {
```

Comment:
If you need an unsigned representation of a `-1` value, you can have it with:
```python
hCurrentProcess = ql.pack(-1)
```
If this is about getting the `MAX_INT` for a given architecture size, then consider:
```python
hCurrentProcess = (1 << ql.arch.bits) - 1
```